src/interfaces/IMorpho.sol

```diff
@@ -123,6 +123,8 @@ interface IMorphoBase {
     /// `liquidate` from being used under certain market conditions.
     /// - A very high price returned by the oracle can make the computation of `maxBorrow` in `_isHealthy` overflow, or
     /// the computation of `assetsRepaid` in `liquidate` overflow.
+    /// @dev The share price of a market with less than 1e4 assets borrowed can be increased by manipulations, to the
```

Comment:
I still don't get it, isn't the attack about inflating the total borrow shares, so deflating / decreasing the share price?

Reply:
you are right, I'm dumb